docs/admin/integrations/jfrog-artifactory.md

```diff
@@ -37,35 +37,36 @@ two type of modules that automate the JFrog Artifactory and Coder integration.
 
 This module is usable by JFrog self-hosted (on-premises) Artifactory as it
 requires configuring a custom integration. This integration benefits from
-Coder's [external-auth][] feature and
-allows each user to authenticate with Artifactory using an OAuth flow and issues
-user-scoped tokens to each user.
+Coder's [external-auth][] feature and allows each user to authenticate with
+Artifactory using an OAuth flow and issues user-scoped tokens to each user.
 
 To set this up, follow these steps:
 
 1. Modify your Helm chart `values.yaml` for JFrog Artifactory to add,
 
-	```yaml
-	artifactory:
-	  enabled: true
-	  frontend:
-	  extraEnvironmentVariables:
-	    - name: JF_FRONTEND_FEATURETOGGLER_ACCESSINTEGRATION
-	      value: "true"
-	  access:
-	  accessConfig:
-	    integrations-enabled: true
-	    integration-templates:
-	      - id: "1"
-	        name: "CODER"
-	        redirect-uri: "https://CODER_URL/external-auth/jfrog/callback"
-	        scope: "applied-permissions/user"
-	```
-	<blockquote class="admonition info">
-
-	Replace `CODER_URL` with your JFrog Artifactory base URL; for example, `coder.mycompany.com`.
-
-	</blockquote>
+   ```yaml
```

Comment:
I ran `make fmt` and it messed it all back up - @matifali has a fix for this coming as `prettier` isn't doing what we want it to in the way we want it to

Reply:
I lied, @matifali doesn't have a fix coming; just faced the same problem